datadog_lambda/tracing.py

```diff
@@ -15,7 +15,10 @@
     TraceContextSource,
 )
 import ddtrace
-ddtrace.config.analytics_enabled = os.environ.get("DD_TRACE_ANALYTICS_ENABLED", "true").lower() == "true"
+
+ddtrace.config.analytics_enabled = (
+    os.environ.get("DD_TRACE_ANALYTICS_ENABLED", "true").lower() == "true"
```

Comment:
Are you able to test the difference before and after? if not yet, let me see if I can find sometime to test it and let's hold off until we have a concrete understanding here.